### DISCUSSION TITLE
How does golang-version data source being used?

### DISCUSSION THREAD

**Discussion by idodod** [renovatebot/renovate#16199] [Request Help] [answered]
I was hoping gomod manager would also use this data source to upgrade the go version in go.mod file but that does not seem to be the case.

Are there any thoughts of adding a matching manager to this datasource?

**Comment by viceice** [comment, marked as answer]
it should be done here
https://github.com/renovatebot/renovate/blob/07b98a90ff6030c9546d56615da9ec51eb7b6e3f/lib/modules/manager/gomod/extract.ts#L49

but i'm not sure if this should be enabled by default, as most users like to have this as minimum supported go version.

@rarkins WDYT

If we aggree, you need to search for an open feature request or open a new feature request if no existing is found.

**Reply by rarkins** [reply]
I'm not sure we have a feature request open (I don't remember one). We should create one if not.

**Reply by HonkingGoose** [reply]
I've created a feature request:

- https://github.com/renovatebot/renovate/issues/16220
